import "proto_struct.proto";

//ÇëÇóÁÄÌìÐÅÏ¢ C->S ÏûÏ¢ŽúÂë:1100
message ClientSendData
{
	optional string SenderName=1;
	optional string TalkMsg = 2;				//ÁÄÌìÐÅÏ¢(25ÖÐÎÄ×Ö·û)
}

//¹ã²¥ÊÀœçÁÄÌìÐÅÏ¢ S->C ÏûÏ¢ŽúÂë:1100
message ServerSendData
{
	optional string Result			= 1;		//œá¹û
	optional string NickName		= 2;		//œÇÉ«Ãû³Æ
	optional string TalkMsg			= 3;		//ÁÄÌìÐÅÏ¢
	optional string TalkTime		= 4;		//ÁÄÌìÊ±Œä
}

//×¢²áÕËºÅ1101
message csRegisterAccount
{
	optional string accountName=1;
	optional string password=2;
	optional string repeatPassword=3;
}

message scRegisterAccount
{
	//1ÕýÈ·0ŽíÎó
	optional bool result=1;
	
}

//µÇÂœÓÎÏ·1102
message csLoginGame
{
	optional string accountName=1;
	optional string password=2;
}

message scLoginGame
{
	optional bool result=1;
}

//»ñÈ¡·þÎñÆ÷ÁÐ±í1103
message csServerList
{
	
}

message scServerList
{
	repeated struct_serverList ServerList=1;
}

//œøÈëÓÎÏ·1104
message csEnterGame
{
	optional int32 ServerId=1;
}

message scEnterGame
{

	optional int32 ServerId=1;
}



